.env.example

```diff
@@ -3,15 +3,32 @@
 # tokens and secrets in this file
 
 # GitHub App ID
-GH_APP_IDENTIFIER=""
+GH_APP_IDENTIFIER=''
 # Github App Webhook Secret to verify requests come from GitHub
-GH_WEBHOOK_SECRET=""
+GH_WEBHOOK_SECRET=''
 # GitHub App Secret Key
 # NOTE: this *must* be on a single line, otherwise it will break
 # reading the secret key.
 GH_APP_SECRET_KEY='-----BEGIN RSA PRIVATE KEY----------END RSA PRIVATE KEY-----'
+# Personal (ie, for your own GitHub account) user access token with the `admin:org`,
+# `admin:org_hook`, `admin: repo_hook`, `project`, `public_repo`, `repo:status`, and
+# `repo_deployment` permissions enabled.
+GH_USER_TOKEN=''
 
 # Slack signing secret to verify requests come from Slack
 SLACK_SIGNING_SECRET=
 # Slack Bot user access token -> "Bot User OAuth Token"
 SLACK_BOT_USER_ACCESS_TOKEN=
+
+# The name of the personal repository in your personal organization being used for development.
+PERSONAL_TEST_REPO='eng-pipes-dev'
+
+# Application-specific configuration
+# Config for the "GitHub Issues Someone Else Cares About" project board
+ISSUES_PROJECT_NODE_ID=''
+STATUS_FIELD_ID=''
+PRODUCT_AREA_FIELD_ID=''
+RESPONSE_DUE_DATE_FIELD_ID=''
+
+# Silence some GCP noise
+DRY_RUN=1
```

Comment:
```suggestion
DRY_RUN=1

```

Reply:
I've flipped the "insert final newline" setting in VSCode, so hopefully this won't happen in the future!